src/componentes/mounted/lista-nominal/ToolBarMounted.tsx

```diff
@@ -0,0 +1,83 @@
+import { ListToolBar, Tooltip, Tag, Icon, Text, SearchBarContainer, Input, Button,  } from '@impulsogov/design-system'
+import { Dispatch, SetStateAction } from 'react';
+
+interface InputProps {
+    value: string;
+    onChange: Dispatch<SetStateAction<string>>;
+}
+
+interface ToolBarMountedProps {
+    updateDate: string | null;
```

Comment:
@dlopes14 recomendo alteração abaixo, esse nome de váriavel fica mais padronizado e deixa-lo como Date é melhor porque vc cuida da formatação na hora de exibir e não na hora de mandar a informação.

Fica mais desacoplado pois por exemplo, caso mude o formato na hora por algum pedido de Produto, vc só edita o format no componente, não precisa mexer em outro lugar.

```suggestion
    updatedAt?: Date;
```

Reply:
@vhbsouza deixei como string porque a resposta do banco que temos hoje é string, caso mude no futuro, seguimos a alteração.

Reply:
@dlopes14 , ao meu ver o argumento do banco não faz sentido atualmente pois nossos dados estão mockados e os dados futuros serão oferecidos no formato que nós especificarmos. Lembre-se, estamos construindo para os novos indicadores e eles usarão os tipos que considerarmos apropriados.

Além de que usando schema validation com zod igual a @tainnaps sugeriu e faremos, podemos facilmente converter o date que vem do banco para qualquer formato que quisermos.

Para além disso, vc não comentou e nem aplicou minha recomendação do `?` para indicar campo opcional ao inves do ` | null`. 

Qual o caminho aqui, na sua visão? Pra mim o ideal seria `?` pois os dados virão da API e não diretamente do Select que faremos no dado então o campo null não virá na requisição REST.